src/pages/signing_official_console/SigningOfficialDarRequests.jsx

```diff
@@ -7,8 +7,10 @@ import { USER_ROLES } from '../../libs/utils'
 import { DarCollectionTable } from '../../components/dar_collection_table/DarCollectionTable'
 import { consoleTypes } from '../../utils/DarCollectionUtils'
 import { useResponsiveDarCollectionColumns } from '../../hooks/useResponsiveDarCollectionColumns'
+import { usePageTitle } from '../../hooks/usePageTitle'
```

Comment:
```suggestion
import { usePageTitle } from 'src/hooks/usePageTitle'
```

And perhaps update the other imports?